### ISSUE TITLE
Key combinations and mouse double clicks

### ISSUE THREAD

**Issue by marc3lsk** [Snoothy/UCR#74]
How to map key combinations like shift+key and mouse double clicks? And something like shift+click? In context.xml, there is KeySubValue, but it is not working for me, for example shift+a:
```
<KeyType>1</KeyType>
<KeyValue>41</KeyValue>
<KeySubValue>29</KeySubValue>
```

**Comment by evilC** [comment]
There are no plans to support temporal mappings such as double-click, as this adds massive complexity and will result in performance degradation for single clicks.  
Multi-key combos are planned to be supported, but will not be done as one mapping - you would create eg a shift mapping which switched state of the profile, thus activating a click input, which maps to an output.

**Comment by Snoothy** [comment]
The Filter feature will be available in release v0.9.0

**Comment by evilC** [comment]
To clarify, this is for input only, support for sending of key combinations as output is currently not on the roadmap
